Data-Structures/src/test/java/linkedList/LinkedListTest.java

```diff
@@ -0,0 +1,101 @@
+package linkedList;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LinkedListTest {
+
+    // Testing for initializing empty linked list with no nodes
+    @Test
+    public void testCreateEmptyLinkedList() {
+        LinkedList test = new LinkedList();
+
+        assertNull("The head of the linked list should be null initially for empty linked list",
+                    test.head);

```

Comment:
I like how you are starting with the basics here and progressively build on that in your tests. Take nothing for granted!